services/static-webserver/client/source/class/osparc/data/model/Node.js

```diff
@@ -1360,6 +1364,33 @@ qx.Class.define("osparc.data.model.Node", {
       pingRequest.send();
     },
 
+    __waitForServiceWebsite: function(srvUrl) {
+      // request the frontend to make sure it is ready
+      let retries = 5;
+      const request = new XMLHttpRequest();
+      const openAndSend = () => {
+        if (retries === 0) {
+          return;
+        }
+        retries--;
+        request.open("GET", srvUrl);
+        request.send();
+      };
+      const retry = () => {
+        setTimeout(() => openAndSend(), 2000);
+      };
+      request.onerror = () => retry();
+      request.ontimeout = () => retry();
```

Comment:
is this retry forever?? i.e. it does not stop after N retries

Reply:
5 times max